### PR TITLE
base64: add Hindi translation

### PR DESCRIPTION
**Pull request by pranavupadhyay123**
- [ ] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [ ] The page description(s) have links to documentation or a homepage.
- [ ] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

## Changes I made

- Fixed the title back to the command name (our linter doesn't support localized page title)
- Translated the placeholders in applicable places. (Note: I am not sure about the plain_file's translation so feel free to update it to a different word)

File: pages.hi/osx/base64.md
Comment (kbdharun) [inline pages.hi/osx/base64.md:4]:
```suggestion
> अधिक जानकारी: <https://www.unix.com/man-page/osx/1/base64/>.
```

**Comment by pranavupadhyay123** [comment]
Hey, maybe the license problem is fixed now. can you merge it now

**Review by kbdharun — APPROVED** [review]
This PR needs 1 more review from a maintainer to be merged.

**Pull request closed without merging**
